editor/src/components/editor/store/insertion-path.ts

```diff
@@ -23,6 +23,8 @@ import { ProjectContentTreeRoot } from '../../assets'
 
 export type InsertionPath = ChildInsertionPath | ConditionalClauseInsertionPath
 
+type ConditionalClauseInsertBehavior = 'replace' | 'wrap-into-fragment'
```

Comment:
Really minor but reads a little clearer to me.
```suggestion
type ConditionalClauseInsertBehavior = 'replace' | 'wrap-with-fragment'
```